scripts/build_deb.sh

```diff
@@ -3,13 +3,13 @@
 
 # build_deb.sh - Build a .deb package for one platform.
 #
-# Syntax:   build_deb.sh <arch> <output directory>
+# Syntax:   build_deb.sh <arch> <output directory> <channel>
 #
 # Examples: scripts/build_deb.sh amd64
 
 set -e
-if [ ! "$#" -eq 2 ]; then
-    echo "Syntax: build_deb.sh <arch> <output directory>"
+if [ "$#" -lt 2 ]; then
+    echo "Syntax: build_deb.sh <arch> <output directory> <channel>"
```

Comment:
you may want to consider using getopts here instead of this approach to do validation on each argument. Not a blocker for the PR, but approaching bash script parameters this way helps new users down the road understand how to use the script faster:


```bash
HELP="Usage: $0 [-v] [-u]
Script for installing the mule cli.
Requires:
    * pip
    * python 3
    * git (And access to algorand/go-algorand-ci repository on github)
Options:
    -v        Version of mule cli (default is '${MULE_VERSION}')
    -u        Authenticate to github (algorand/go-algorand-ci repository) with user credentials (default is over ssh)
"

while getopts ":v:uh" opt; do
  case ${opt} in
    v ) MULE_VERSION=$OPTARG
      ;;
    u ) USER_AUTH="True"
      ;;
    h ) echo "${HELP}"
        exit 0
      ;;
    \? ) echo "${HELP}"
        exit 2
      ;;
  esac
done
```

Reply:
I've used a similar construction for other scripts, but I didn't for this because other pre-existing scripts are using this and I wanted to leave as light a footprint as I could.

Also, I feel that depending on how the script is used it's better to use positional arguments, especially for automation.

Reply:
I agree that including parameter names helps clarify and ensure proper use.

Reply:
This is not intended to be a cli tool.